_variables.scss

```diff
@@ -34,3 +35,20 @@ $fontsizeL:      16px;
 $fontsizeXL:     18px;
 $fontsizeXXL:    22px;
 $fontsizeXXXL:   24px;
+
+
+// Various lines
+// can be used as borders, underlines, separators
+// https://projects.invisionapp.com/share/8CAS4KBDB#/screens/82068093_StyleguideM1-VisualElements-Lines
+
+// line weigh
+$lineThin:        1px;
+$lineThick:       2px;
+
+// line style
+$lineDotted:      dotted;
```

Comment:
can you advise me a better solution to centralize these line styles?
https://projects.invisionapp.com/share/8CAS4KBDB#/screens/82068093_StyleguideM1-VisualElements-Lines